README.md

```diff
@@ -1,17 +1,91 @@
+  </a>
+
+  <h3 align="center">dgit</h3>
+
+  <p align="center">
+    <b>dgit</b> is an opensource project built by Quorum Control which combines
+    the power of <br>git, the Tupelo DLT and Skynet from Sia.  <br>
+    <b>dgit</b> uses decentralized ownership and storage to make it trivial to
+    create a decentralized mirror of your project.<br>
+    <b>dgit</b> accomplishes this without changing your github workflow.<br>
+  </p>
+</p>
+
+### Built With
+
+* [Git](https://git-scm.com/)
+* [Tupelo DLT](https://docs.tupelo.org/)
+* [Skynet](https://siasky.net/)
+
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+
+- [Getting Started](#getting-started)
+  - [Installation](#installation)
+  - [Usage](#usage)
+  - [Building](#building)
+- [Contributing](#contributing)
+- [License](#license)
+- [Contact](#contact)
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+With three simple steps you can create a decentralized mirror of your existing github project.
+All changes will be automatically propogated to the mirror version and the git services you depend on will be there when you need them.
+
+### Installation
+A quick install using brew gets us started: 
+`brew tap quorumcontrol/dgit && brew install dgit` <br>
+Or skip the cask and directly install with: 
+`brew install quorumcontrol/dgit/dgit`
 
 ### Usage
-Protocol is registered as `dgit`, so origin should look like:
-`git remote add origin dgit://quorumcontrol/tupelo`
+Next you are going to run the init command in each repo you want to make decentralized:
+`dgit init`
 
-Replacing `quorumcontrol/tupelo` with any repo name
+This command does three things.<br>
+First it sets the appropriate remote urls in gits .config file.<br>
+Second, it creates a [ChainTree](https://docs.tupelo.org/docs/chaintree.html) which gets signed by the Tupelo DLT to specify ownership of the decentralized repo.<br>
+Third, it stores that repo on Skynet the decentralized storage solution from Sia. 
 
-Then proceed with normal git commands
+From there you can proceed with normal git commands.
+The decentralized mirror will stay in sync as you make changes.
+If you ever want to pull from the mirror you can specify the mirror with a "dgit:".<br>
+As an exmaple:
+`git clone dgit:\\your_username\repo_name`
 
 ### Building
 - Clone this repo.
 - Run `make`. Generates `./git-remote-dgit` in top level dir.
 
-### Installation
-- Run `make install`. Copies `git-remote-dgit` to your $GOPATH/bin dir, so add that to your path if necessary.
\ No newline at end of file
+<!-- CONTRIBUTING -->
+## Contributing
+
+Contributions are what make the open source community such an amazing place to be learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+
+1. Fork the Project
+2. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
+3. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
+4. Push to the Branch (`git push origin feature/AmazingFeature`)
+5. Open a Pull Request
+
```

Comment:
We should develop a dgit-only workflow for this as an early post-hackathon addition. :)

Reply:
Agreed, nice idea!